crates/bevy_app/src/app.rs

```diff
@@ -18,13 +18,9 @@ use bevy_utils::tracing::info_span;
 bevy_utils::define_label!(AppLabel);
 
 #[allow(clippy::needless_doctest_main)]
-/// Containers of app logic and data
+/// An ECS application that wraps a [`World`], a runner function, and a [`SubApp`] collection.
```

Comment:
`SubApp` is private, so this should be formatted as sub-`App`

Reply:
Seems like there is a lot of unrelated doc changes in this PR. Would there be any value in splitting these out to a separate PR?

Reply:
Yeah, probably. Anything that makes the line count less intimidating.